R/gather.R

```diff
@@ -33,9 +33,10 @@
 #' expression, you can unquote them with the tidy eval operator
 #' `!!`. This operator evaluates its argument in the context and
 #' inlines the result in the surrounding function call. For instance,
-#' `c(x, !! x)` selects the `x` column within the data frame and the
-#' column referred to by the object `x` defined in the context (which
-#' can contain either a column name as string or a column position).
+#' `c(x, !! a)` selects the `x` column within the data frame and the
```

Comment:
I think this section needs a full revamp to use `any_of()`/all_of()` instead of `!!`. Do you want to take a stab at that or leave it up to us?

Reply:
That makes sense, I'll take a stab at it.